wp-content/themes/missilethreat/archive.php

```diff
@@ -43,26 +45,137 @@
 
 </header>
 
-<div class='archive'>
+<div class='archive__content'>
 
-  <section class="archive__content">
-    
   <?php
+  if ( is_post_type_archive('missile') ) { ?>
+
+    <section class="actors">
+      <div class="archive__desc">Ballistic missiles, cruise missiles, rockets, artillery, and mortars (RAM), and even maneuvering hypersonic boost glide delivery systems now form the complicated 21st century strike complex with which U.S., allied, and partner nations must contend. Organized by country, the following represents a growing collection of information on global missile systems, with illustrations and up-to-date information on their capabilities and history.</div>
```

Comment:
It would be worth considering making this a widget, or making an ACF for [this archive page](https://wordpress.org/plugins/acf-archive/) instead of hardcoding this text so they can easily update it later if they need to.

It should also be a `<p>` tag, not a `<div>`.